client/dist/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Please update this to follow the following form:
```
<html lang="en">
<head>
<meta charset="UTF-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
...
```

Reply:
Done!